fhi_users/auth/auth_utils.py

```
@@ -245,3 +244,28 @@ def create_user(
+        .replace("+", "")
+        .replace("(", "")
+        .replace(")", "")
+        .replace(",", "x")  # Some folks use comma for extensions
+    )
+
+    # Check that the remaining string only contains digits and 'X' or 'x'
+    if (
+        not all(char.isdigit() or char == "X" or char == "x" for char in cleaned_number)
+        or len(cleaned_number) < 10
+    ):
+        raise ValidationError(
```

Ruff (0.8.2)

Undefined name `ValidationError`

(F821)